.devcontainer/podman/devcontainer.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "podman ludeeus/integration_blueprint",
```

Comment:
This is not something I use, so Im not willing to maintain a separate instruction set for it.

Reply:
![image](https://github.com/user-attachments/assets/10adcf29-a42c-42d7-aa72-eab70e973b2e)